content/en/docs/reference/api/account-api/_index.md

```diff
@@ -475,6 +500,120 @@ $ curl -s -H "Authorization: Bearer ${AT}" \
     ...
 ```
 
+The following example returns the 5 first users, who has one of the aformentioned filtering attributes containing "@iam" in a descing order according to the names (implicit sorting attribute is name)
```

Comment:
I would rephrase it like this: The following example returns the first 5 users who have one of the above filter attributes containing “@iam” in descending order by name (the implicit sorting attribute is name).